src/content/docs/change-tracking/change-tracking-introduction.mdx

```diff
@@ -36,27 +36,27 @@ Change tracking has a number of features that help you gauge the effects of chan
 
 Our goal is to provide full-platform coverage for markers on charts, but some of the New Relic curated experiences don't yet feature interactive markers on all charts. In the meantime, you can still record changes on relevant entity types and build your own experience around that type using a custom dashboard.
 
 ### Quickly answer the question “What changed?” [#what-changed]
 
 Once you start [capturing deployments](#start-tracking) and other changes to entities in your system, you'll automatically see interactive markers drawn on related charts in APM, mobile, browser, service level management, custom dashboards, and elsewhere. You'll also see entity sidebar options related to change tracking (for example, **Deployments**) popping up across New Relic.
 
-### Lower your mean time to repair (MTTR) [#mttr]
+### Lower your mean time to resolution (MTTR) [#mttr]
 
 Change tracking has powerful interfaces to help you quickly understand the impact a change has had on your system. Here are some tasks change tracking can help with:
 
-* Want to see how many regressions or new error groups we detected since a change?
+* Want to see how many regressions or new error groups we detected since a change? For more about regressions, see [errors inbox](/docs/errors-inbox/errors-inbox).
 * Want to see what anomalous activity we identified since a change?
 * Need to run a head-to-head comparison of deployments to see if things are getting better or worse?
 
 You can do these tasks (and more) with our new change tracking UIs. Read about the change tracking experience in [How to view and analyze your changes in New Relic](/docs/change-tracking/change-tracking-view-analyze/).
 
-### All deployment and change data is now in one place [#single-source-truth]
+### All deployment and change data is now stored in one place [#single-source-truth]
 
-Every change you capture via change tracking APIs or integrations is stored in the New Relic database (NRDB). You can query deployment and change data using NRQL and NerdGraph. For examples of how to query for data about changes you have captured see [Track changes using NerdGraph](/docs/change-tracking/change-tracking-graphql/#query-events).
+Every change you capture via change tracking APIs or integrations is stored in the New Relic database (NRDB). You can query deployment and change data using NRQL and NerdGraph. For examples of how to query for data about changes you have captured see [Query deployments data](/docs/change-tracking/change-tracking-view-analyze/#query-deployments).
```

Comment:
I don't know if it's just being flaky, if it's my browser, or what, but for some reason this anchor tag doesn't jump to the right section. If it works for you, that's good enough for me.